tests/src/FunctionalJavascript/ParagraphsFeaturesSplitTextTest.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+namespace Drupal\Tests\paragraphs_features\FunctionalJavascript;
+
+use Drupal\editor\Entity\Editor;
+use Drupal\filter\Entity\FilterFormat;
+
+/**
+ * Tests the paragraph text split feature.
+ *
+ * @group paragraphs_features
+ */
+class ParagraphsFeaturesSplitTextTest extends ParagraphsFeaturesJavascriptTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
```

Comment:
The modules, which are present in ParagraphsFeaturesSplitTextTest are not needed here